src/population.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+class Population():
+    def __init__(self, solutions, fitness):
+        self.solutions = solutions
+        self.fitness = fitness
+
+    def initial_population(self, population_size, test_size):
+        population = list()
+        for i in range(population_size):
+            test = list()
```

Comment:
There is a specific TestCase class for the test, so probably you should create an instance of that class.